cmd/skopeo/list_tags.go

```diff
@@ -28,6 +29,20 @@ type tagsOptions struct {
 	retryOpts *retry.RetryOptions
 }
 
+var transportHandlers = map[string]func(ctx context.Context, sys *types.SystemContext, opts *tagsOptions, userInput string) (repositoryName string, tagListing []string, err error){
+	docker.Transport.Name():  listDockerRepoTags,
+	archive.Transport.Name(): listDockerArchiveTags,
+}
+
+// getSupportTransport return all the supported transports
+func getSupportTransport(joinStr string) string {
+	supportTranspors := make([]string, 0, len(transportHandlers))
```

Comment:
```suggestion
// getSupportedTransports returns all the supported transports
func getSupportedTransports(joinStr string) string {
	supportedTransports := make([]string, 0, len(transportHandlers))
```

---

Absolutely non-blocking: Alternatively, the function could just be called `supportedTransports`, and the local variable maybe just `res`.